adafruit_turtle.py

```diff
@@ -743,30 +751,32 @@ def towards(self, x1, y1=None):
         :param y: a number if x is a number, else None
 
         """
-        if y1 is None:
-            y1 = x1[1]
-            x1 = x1[0]
-        x0, y0 = self.pos()
+        yn: float = x1[1] if y1 is None else y1  # type: ignore
+        xn: float = x1[0] if y1 is None else x1  # type: ignore
+        p = self.pos()
+        x0, y0 = (p[0], p[1])
```

Comment:
I'm curious, was there a specific reason these were broken up into do different steps?  Not an issue, just wondering why the function call and unpacking happen separately.